### PR TITLE
Add posibility to set custom SUT name in svirt backend

### PR DESCRIPTION
**Pull request by mimi1vx**
It uses VIRSH_GUESTNAME and VIRSH_INSTANCE, defaults to
"openQA-SUT-1"

### CODE REVIEW

File: consoles/sshVirtsh.pm [hunk 1]
```diff
@@ -34,8 +34,8 @@ sub new {
     my $self = $class->SUPER::new($testapi_console, $args);
 
     $self->instance(get_var('VIRSH_INSTANCE', 1));
-    # default name
-    $self->name("openQA-SUT-" . $self->instance);
+    # default name is openQA-SUT-1
+    $self->name(get_var('VIRSH_GUESTNAME', "openQA-SUT-") . $self->instance);
```
Comment (aaannz) [inline consoles/sshVirtsh.pm]: I have a mixed feelings about the separator (-) to be a part of VIRSH_GUESTNAME.

@mimi1vx What separator are those test machines using? Can the dash be hardcoded?
```perl
$self->name(get_var('VIRSH_GUESTNAME', 'openQA-SUT') . '-' . $self->instance);
```
or
```perl
$self->name(sprintf('%s-%u', get_var('VIRSH_GUESTNAME', 'openQA-SUT'), $self->instance));
```

Comment (mimi1vx) [inline consoles/sshVirtsh.pm]: why hardcode any character ? waht about 'underscore' or 'dot' ?

Comment (aaannz) [inline consoles/sshVirtsh.pm]: I just don't like VIRSH_GUESTNAME to have trailing separator.
But true, we don't have to use any separator and it's more future-proof not to hardcode anything. Disregard my comments.

**Comment by coolo** [comment]
I added attach_to_running for this functionality.

**Comment by coolo** [comment]
and just as @aaannz I don't like that GUESTNAME and INSTANCE are mangled. If you already know the  domain name, then why split it artificually

**Comment by mimi1vx** [comment]
@coolo oh, I'm blind ... , please forget about this PR

**Pull request closed without merging**